better_bookmarks/js/modal.js

```diff
@@ -17,17 +17,11 @@ var usableU;
 // This will be the tabs description
 //var usableD;
 
-//Get access to
-chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
-    var tab = tabs[0];
-    var title = tab.title;
-    var url = tab.url;
-    //var body = tab.title;   // document.queryselector('body').innertext
-    usableT = title;
-    usableU = url;
-    //usableB = body;
-    autofiller(usableT, usableU);
-})
+chrome.storage.sync.get(["title", "url"], function(result) {
+    usableT = result.title;
```

Comment:
What's going on here?  Why is the line immediately above, line 21, and the two lines below, lines 22 and 23, indented at a different level?  Everything here should be indented at the same level, which is 4 spaces.

Reply:
It doesn't look like you fixed this.  Fix the indentation by placing everything in this block at an indentation level of 4 spaces.

Reply:
Okay, now I see the fix; resolved.